src/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/Real-Dev-Squad/gopher-cloud-service/src/routes"
+)
+
+func main() {
+	value, keyExists := os.LookupEnv("env")
+	port := strconv.Itoa(8000)

```

Comment:
Can we read this from config? viper itself

Reply:
Viper will be included in future PRs then i will read from config that is why I kept it as a variable